integration/nydus/nydus_tests.sh

```diff
@@ -86,13 +86,13 @@ function config_kata() {
 	sudo mkdir -p /etc/kata-containers
 	if [ -f "$SYSCONFIG_FILE" ]; then
 		need_restore_kata_config=true
-		sudo cp -a "${SYSCONFIG_FILE}" "${kata_config_backup}"
```

Comment:
@fidencio here it saves the file to a backup that is restored later. Either you preserve this backup mechanism or change the logic to switch the link to the default config.

Reply:
Solved, thanks!

Reply:
It is still different: sudo cp -a" ${SYSCONFIG_FILE}" "${kata_config_backup}"

Reply:
Sorry, my bad, fixed now.